lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -300,13 +300,23 @@ sub restart {
     $self->render(json => {result => $res, test_url => \@urls});
 }
 
+# Used for both apiv1_cancel and apiv1_cancel_jobs
 sub cancel {
     my ($self) = @_;
-    my $jobid = int($self->param('jobid'));
+    my $jobid = $self->param('jobid');
```

Comment:
It showed "use of unitialized value in function int()" warning when called without job id.